### PR TITLE
Minor refactoring and bug fix

### PR DESCRIPTION
**Pull request by kirbo**

### CODE REVIEW

File: argonOne/run.sh [hunk 1]
```diff
@@ -33,7 +33,7 @@ fcomp() {
 
 
 fanSpeedReport(){
-   percent=${1};
+   fanPercent=${1};
```
Comment (kirbo) [inline argonOne/run.sh:36]: Making variables more consistent, since there was lots of inconsistency

File: argonOne/run.sh [hunk 2]
```diff
@@ -54,40 +54,40 @@ fanSpeedReport(){
...
-  percent=${1}
+  fanPercent=${1}
   level=${2}
   name=${3}
-  percentHex=${4}
-  temp=${5}
-  CorF=${6}
+  temp=${4}
+  CorF=${5}
+  fanPercentHex=$(printf '%x' "${fanPercent}")
```
Comment (kirbo) [inline argonOne/run.sh:75]: There was no reason to pass the `fanPercentHex` as an argument, as it is used only here. Therefore you can rather just do it here instead

File: argonOne/run.sh [hunk 3]
```diff
@@ -185,13 +185,12 @@ until false; do
         fanPercent=100;
         ;;
     esac
-    fanPercentHex=$(printf '%x' ${fanPercent})
-    action "${curPosition}" "${fanPercent}" "${curPositionName}" "${fanPercentHex}" "${cpuTemp}" "${CorF}"
+    action "${fanPercent}" "${curPosition}" "${curPositionName}" "${cpuTemp}" "${CorF}"
```
Comment (kirbo) [inline argonOne/run.sh:188]: Here you had a bug. You passed a `curPosition` into `percent` and vice versa. This fixes the issue https://github.com/adamoutler/HassOSArgonOneAddon/issues/19

**Comment by adamoutler** [comment]
Thanks.  It's good to have contributions. I never go past 1%

**Pull request merged**